common/Cargo.toml

```diff
@@ -6,10 +6,14 @@ edition = "2021"
 [dependencies]
 anyhow = { workspace = true }
 bincode = { workspace = true }
-jupiter-core = { workspace = true }
 rust_decimal = "^1.0"
 solana-program = { workspace = true }
 spl-token = { workspace = true }
 stakedex_deposit_sol_interface = { workspace = true }
 stakedex_interface = { workspace = true }
+jupiter-amm-interface = { workspace = true }
 thiserror = { workspace = true }
+
+serde_json = "1.0.95"
+serde = "1.0.159"
```

Comment:
i think we want to move this to workspace to make sure all the packages use the same vers of `serde`, but i can just make this change after this is merged